Backbone.API/Configuration/ChallengesConfiguration.cs

```diff
@@ -21,5 +21,9 @@ public class SqlDatabase
 {
     [Required]
     [MinLength(1)]
-    public string ConnectionString { get; set; } = "";
+    public string Provider { get; set; } = string.Empty;
```

Comment:
I just had an idea. We should use the `RegularExpression` attribute to allow only valid providers. 

```
[RegularExpression("SqlServer|Postgres")]
```

Can you do the same for `EventBusConfiguration#Vendor` and all `BlobStorageConfiguration#CloudProvider` please?